Makefile

```diff
@@ -66,7 +66,8 @@ vendor:
 
 .PHONY: install
 install:
-	install -D -m755 $(BUILD_DIR)/$(NAME) $(BIN_DIR)
+	cp $(BUILD_DIR)/$(NAME) $(BIN_DIR)
+	chmod 755 $(BIN_DIR)$(NAME)
```

Comment:
```suggestion
	chmod 755 $(BIN_DIR)/$(NAME)
```
Just to be extra cautious.

Reply:
This now outputs path with `//` because `BIN_DIR` already has trailing `/`

Reply:
Can that cause any side effects?

Reply:
`//` evaluates to `/`, so we should be safe